resources/standards-data/unicode-character-database/build.sh

```diff
@@ -0,0 +1,68 @@
+#!/usr/bin/env bash
+#
+# Compile our sourcemap-path remapping module for use by Web builds, releases, etc.
+#
+## START STANDARD BUILD SCRIPT INCLUDE
+# adjust relative paths as necessary
+THIS_SCRIPT="$(readlink -f "${BASH_SOURCE[0]}")"
+. "$(dirname "$THIS_SCRIPT")/../../../resources/build/builder.inc.sh"
+## END STANDARD BUILD SCRIPT INCLUDE
+
+. "$KEYMAN_ROOT/resources/build/minimum-versions.inc.sh"
+
+################################ Main script ################################
+
+builder_describe \
+  "Downloads Unicode data consumed by various Keyman platforms." \
+  clean configure build
+
+builder_describe_outputs \
+  configure     /resources/standards-data/unicode-character-database/UnicodeData.txt \
+  build         /resources/standards-data/unicode-character-database/UnicodeData.txt
+
+builder_parse "$@"
+
+# Used by Developer
+BLOCKS_SRC_HREF="https://www.unicode.org/Public/$KEYMAN_VERSION_UNICODE/ucd/Blocks.txt"
+BLOCKS_SRC_LOCAL="./Blocks.txt"
+
+UNICODE_DATA_SRC_HREF="https://www.unicode.org/Public/$KEYMAN_VERSION_UNICODE/ucd/UnicodeData.txt"
+UNICODE_DATA_SRC_LOCAL="./UnicodeData.txt"
+
+# Used by common/models/wordbreakers for the default Unicode wordbreaker.
+WORDBREAK_PROP_SRC_HREF="https://www.unicode.org/Public/$KEYMAN_VERSION_UNICODE/ucd/auxiliary/WordBreakProperty.txt"
+WORDBREAK_PROP_SRC_LOCAL="./WordBreakProperty.txt"
+
+EMOJI_DATA_SRC_HREF="https://www.unicode.org/Public/$KEYMAN_VERSION_UNICODE/ucd/emoji/emoji-data.txt"
+EMOJI_DATA_SRC_LOCAL="./emoji-data.txt"
```

Comment:
One idea I had, but chose not to implement (yet): what if we replicated the Unicode subpath structure here?

So, `WORDBREAK_PROP_SRC_LOCAL="./auxiliary/WordBreakProperty.txt"`, etc.  With that pattern, we could simply specify a shell-array of subpaths to download, rather than needing a variable for each one.